### ISSUE TITLE
Support get query that doesn't require an id

### ISSUE THREAD

**Issue by michaelst** [ash-project/ash_graphql#12]
This below will error if you try to query without an id even though we don't require one.

``` 
actions do
  read :current_user do
    filter id: actor(:id)
  end
end

graphql do
  type :post

  queries do
    get :current_user, :current_user
  end
end
```

I would be happy to do the PR for this if you let me know how you would want it implemented, maybe a new verb? Because I could see wanting to keep ID! as a requirement for the queries where it is required.

**Comment by zachdaniel** [comment]
I think a new verb would work well, specifically a `read_one` to contrast a `get`. `get` uses an identity whereas `read_one` simply expects the query to return one result (using `Api.read_one/2`)

**Comment by zachdaniel** [comment]
I'll leave a note here shortly with some implementation tips, would be very grateful to have you PR it.

**Comment by zachdaniel** [comment]
So step 1 will be to add an entity to the DSL. You can model it off of `get`, but it just won't take an `identity` option.

https://github.com/ash-project/ash_graphql/blob/master/lib/resource/resource.ex#L7


You'll need to add code to determine the query type: (same as `get`)

https://github.com/ash-project/ash_graphql/blob/master/lib/resource/resource.ex#L562

You'll need to configure it to accept the `read_args`, but not the identity like `get` does:

https://github.com/ash-project/ash_graphql/blob/master/lib/resource/resource.ex#L580

And then you'll need to add a resolver for `:read_one`:

https://github.com/ash-project/ash_graphql/blob/master/lib/graphql/resolver.ex#L6

I think that should be it.

**Comment by michaelst** [comment]
https://github.com/ash-project/ash_graphql/pull/13 not sure if there is any testing or other documentation you would want me to add on this

**Comment by zachdaniel** [comment]
If you could add a test in `read_test.exs` that would be great :D

**Comment by michaelst** [comment]
test added

**Comment by zachdaniel** [comment]
Released in `0.14.0`
